### PR TITLE
Initial Linux support

### PR DESCRIPTION
**Pull request by technobaboo**
All basic functions of Open Brush work in Linux now, so this PR is just to add support. Some credit goes to Babouchot for their contributions (see #4)

### CODE REVIEW

File: Assets/Editor/BuildTiltBrush.cs [hunk 1]
```diff
@@ -173,13 +175,17 @@ class PostBuildInfo {
...
       Menu.SetChecked(kMenuPlatformWindows, false);
+      Menu.SetChecked(kMenuPlatformLinux, false);
       Menu.SetChecked(kMenuPlatformOsx, false);
       Menu.SetChecked(kMenuPlatformAndroid, false);
 
       switch (value) {
         case BuildTarget.StandaloneWindows64:
           Menu.SetChecked(kMenuPlatformWindows, true);
           break;
+		case BuildTarget.StandaloneLinux64:
```
Comment (mikeage) [inline Assets/Editor/BuildTiltBrush.cs:186]: Are these tabs instead of spaces?

File: Assets/Scripts/App.cs [hunk 2]
```diff
@@ -1919,12 +1923,16 @@ public enum AppState {
       Application.OpenURL(url);
     }
 #else
-    // Something about the url makes OpenURL() not work on OSX, so use a workaround
-    if (Application.platform == RuntimePlatform.OSXEditor ||
-        Application.platform == RuntimePlatform.OSXPlayer) {
-      System.Diagnostics.Process.Start(url);
-    } else {
-      Application.OpenURL(url);
+    // Something about the url makes OpenURL() not work on OSX or Linux, so use a workaround
```
Comment (mikeage) [inline Assets/Scripts/App.cs:1926]: This comment doesn't match the update; did you mean to add the Linux RuntimePlatforms to the cases?

File: Assets/Scripts/Sharing/DriveAccess.cs [hunk 3]
```diff
@@ -99,6 +99,10 @@ private class ParentNotFound : Exception {
       case RuntimePlatform.OSXEditor:
         Debug.LogError("Host id not implemented for macOS");
         return "macOS-unknown";
+	  case RuntimePlatform.LinuxPlayer:
```
Comment (mikeage) [inline Assets/Scripts/Sharing/DriveAccess.cs:102]: tabs again?

**Comment by technobaboo** [comment]
I fixed all of these :D

**Comment by mikeage** [comment]
Thanks. One other thing, but possibly dependent on #24  -- the target in the python script is StandaloneLinuxUniversal, not StandaloneLinux64. That'll probably need to be updated too, but maybe best to do it there (or maybe it depends on which PR gets merged first)

**Comment by mikeskydev** [comment]
I was able to build and run the output, just noting that I had to follow this guide https://github.com/ValveSoftware/SteamVR-for-Linux#runtime-requirements to be able to run it. Is it worth including a Linux section in README to highlight this? I'm not sure if it was just specific to my setup (Ubuntu 20.04 LTS). Will now test from Windows side to make sure nothing breaks.

**Review by mikeskydev — APPROVED** [review]
LGTM! No issues Windows side.

**Comment by mikeskydev** [comment]
Thanks!

**Comment by mikeage** [comment]
@technobaboo , I just wanted to let you and @Babouchot know that we now have Linux builds included as part of the automated pre-release builds! 

https://github.com/icosa-gallery/open-brush/pull/76

**Comment by technobaboo** [comment]
yaaaaay, so excited to get into more cool stuffs, maybe even hand tracking?

**Comment by Babouchot** [comment]
> 
> 
> @technobaboo , I just wanted to let you and @Babouchot know that we now have Linux builds included as part of the automated pre-release builds!
> 
> #76

Nice one !

**Pull request merged**